met-web/src/components/engagement/listing/AdvancedSearch/SearchComponent.tsx

```diff
@@ -0,0 +1,420 @@
+                                        onChange={handleStatusFilterChange}
+                                        value={EngagementDisplayStatus[EngagementDisplayStatus.Upcoming]}
+                                        checked={
+                                            statusFilter[EngagementDisplayStatus[EngagementDisplayStatus.Upcoming]]
+                                        }
+                                    />
+                                }
+                                label={
+                                    <MetParagraph>
+                                        Published - {EngagementDisplayStatus[EngagementDisplayStatus.Upcoming]}
+                                    </MetParagraph>
+                                }
+                            />
+                            <FormControlLabel
+                                control={
+                                    <Checkbox
+                                        size="small"
+                                        id={EngagementDisplayStatus.Open.toString()}
+                                        name={EngagementDisplayStatus[EngagementDisplayStatus.Open]}
+                                        onChange={handleStatusFilterChange}
+                                        value={EngagementDisplayStatus[EngagementDisplayStatus.Open]}
+                                        checked={statusFilter[EngagementDisplayStatus[EngagementDisplayStatus.Open]]}
+                                    />
+                                }
+                                label={
+                                    <MetParagraph>{EngagementDisplayStatus[EngagementDisplayStatus.Open]}</MetParagraph>
+                                }
+                            />
+                            <FormControlLabel
+                                control={
+                                    <Checkbox
+                                        size="small"
+                                        id={EngagementDisplayStatus.Closed.toString()}
+                                        name={EngagementDisplayStatus[EngagementDisplayStatus.Closed]}
+                                        onChange={handleStatusFilterChange}
+                                        value={EngagementDisplayStatus[EngagementDisplayStatus.Closed]}
+                                        checked={statusFilter[EngagementDisplayStatus[EngagementDisplayStatus.Closed]]}
+                                    />
+                                }
+                                label={
+                                    <MetParagraph>
+                                        {EngagementDisplayStatus[EngagementDisplayStatus.Closed]}
+                                    </MetParagraph>
+                                }
+                            />
+                        </FormGroup>
+                    </Grid>
+                </Grid>
+                {!isMediumScreen ? (
```

Comment:
If mobile and desktop designs are different can we separate them into different components